### PR TITLE
Jw/add saml

### PR DESCRIPTION
**Pull request by JenniWhitman**
# What are the relevant tickets?
Updates #4 

# Description (What does it do?)
Moving auth to server directory instead

# How can this be tested?
Push to QA and build and see if we get not-Google auth.

### CODE REVIEW

File: server/userAuth.js [hunk 1]
```diff
@@ -81,6 +104,18 @@ router.get('/auth/redirect', passport.authenticate(authStrategy, {failureRedirec
   res.redirect(req.session.authRedirect || formatUrl('/'))
 })
 
+router.get('/metadata', function (req, res) {
+  if (isSamlAuth) {
+    res.type('application/xml')
+    res.send((samlStrategy.generateServiceProviderMetadata(
+      process.env.SAML_MD_DECRYPTION_CERT,
```
Comment (shaidar) [inline server/userAuth.js]: I would change this to `SAML_SP_ENCRYPTION_CERT`

File: server/userAuth.js [hunk 2]
```diff
@@ -81,6 +104,18 @@ router.get('/auth/redirect', passport.authenticate(authStrategy, {failureRedirec
...
+      process.env.SAML_MD_SIGNING_CERT
```
Comment (shaidar) [inline server/userAuth.js]: I would change this to SAML_SP_SIGNING_CERT

Comment (JenniWhitman) [reply server/userAuth.js]: Thank you!

File: server/userAuth.js (hunk 1 above)
Comment (JenniWhitman) [reply server/userAuth.js]: Thank you!

**Pull request merged**